types/googlemaps/reference/map.d.ts

```diff
@@ -1,3 +1,9 @@
+// TODO: #50253 remove after that most libraries will go to MapMouseEvent
+/**
+ * @internal
+ */
+type GoogleMapsNativeMouseEvent = MouseEvent;
```

Comment:
This is the only way I know how to use a global type whose name is the same as the name from the local namespace.